### ISSUE TITLE
Failure To Downlod Mods Correctly

### ISSUE THREAD

**Issue by DPxKnightxZero** [superzanti/ServerSync#17]
Okay my players and I now realize something no more than one person at a time can download anything. Its imperative that there is at least 5 slots at a time. I realize this will slow downloading time down but at least people won't just get server rejections!

**Comment by superzanti** [comment]
There are infinite slots currently available.

**Comment by DPxKnightxZero** [comment]
Okay well randomly when a player attempts to download the files from the server this occurs.
![2015-06-12_18 46 31](https://cloud.githubusercontent.com/assets/10351221/8141871/97f74532-1133-11e5-896a-b910cf8d4f90.png)

**Comment by DPxKnightxZero** [comment]
https://gist.github.com/DPxKnightxZero/d5ed30fc61895127c2f7

**Comment by DeMaoser** [comment]
Thinking it could be an issue with ServerSync or possibly something with the server host?

**Comment by DPxKnightxZero** [comment]
Proof Server Is On
http://minecraftservers.org/server/242412

**Comment by killerkdawg** [comment]
yes i'm having same issue, keeps saying failure to connect due to saying server may be down but it's up.

**Comment by killerkdawg** [comment]
![moderrorminecraft](https://cloud.githubusercontent.com/assets/12867646/8142310/7ceb9460-1146-11e5-8559-dabea777bb1a.png)
my error

**Comment by superzanti** [comment]
@killerkdawg 

You're on the same server as DPxKnightxZero, so it's probably just a server thing.

However, if you're willing to use linux, my mod is working great for it. It wasn't until recently that I discovered that It has some bugs on windows.

**Comment by DPxKnightxZero** [comment]
Why would we use an entire os just to play the game?.? Why not just dualboot windws 7 or something?.? You could always run it off a usb too!

**Comment by DeMaoser** [comment]
Upon further inspection, it has nothing to do with the server host, the server host has a 500Mbps Upload speed, so if everything worked correctly, the files should download in under 3 minutes.

**Comment by superzanti** [comment]
@DPxKnightxZero 

I've been patient with you for quite a while, and I was okay debugging just for you in my free time, but bringing all your friends into the thread to try and make me move a little faster, is in fact, just going to slow me down.

Be patient. I only released this mod a few days ago. I know there are bugs. If you don't want there to be bugs, and you are desperate to use it, everyone has to use linux. I am using some system calls that only work in linux.

Please do not start another issue until I release the next version.
